src/ui/2d/token.js

```diff
@@ -60,25 +66,96 @@ export class Token extends React.Component {
     onMouseOut: PropTypes.func,
     children: PropTypes.element,
     animationDuration: PropTypes.number,
+    draggable: PropTypes.bool,
+    shouldDrag: PropTypes.func,
+    onDrag: PropTypes.func,
+    onDrop: PropTypes.func,
+    svgRef: PropTypes.object,
   };
 
   static defaultProps = {
     animationDuration: 750,
     template: Square,
   };
 
+  _startDrag = e => {
```

Comment:
Have you taken a look at Card / Deck's drag-n-drop implementation?  Any reason to not do the same here?

I like the fact that this doesn't use an external dependency.  Maybe we should also move over the card / deck implementation to do the same?

Reply:
I had a look before I implemented this, but it seems to me that the library we are using for the cards (react-dragtastic) does not support mobile (see https://github.com/chrisjpatty/react-dragtastic/issues/36 ...), which is a deal breaker for my application. Also, I wasn't sure if it would support internal SVG elements, because all examples are using divs and other HTML elements. 

The implementation for this PR works both in mobile and SVG.

Reply:
OK, I'll try to move over the react-dragtastic implementation to be more in line with this then.